PhysicsTools/PatAlgos/python/cleaningLayer1/tauCleaner_cfi.py

```diff
@@ -8,7 +8,7 @@
         'tauID("decayModeFinding") > 0.5 &'
         ' tauID("byLooseCombinedIsolationDeltaBetaCorr3Hits") > 0.5 &'
         ' tauID("againstMuonTight3") > 0.5 &'
-        ' tauID("againstElectronVLooseMVA6") > 0.5'
+        ' tauID("againstElectronVLooseMVA5") > 0.5'
```

Comment:
@mmarionncern 
it would be better to ```git rebase -i``` and remove the commit that edits this file